resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentVidServiceImpl.java

```diff
@@ -320,8 +320,8 @@ private VidGeneratorResponseDto vidDeactivator(VidRevokeRequestDTO requestDto, S
 						+ JsonUtils.javaObjectToJsonString(request));
 
 		try {
-			response = (ResponseWrapper) residentServiceRestClient.postApi(
-					env.getProperty(ApiName.IDAUTHREVOKEVID.name()), MediaType.APPLICATION_JSON, request,
+			response = (ResponseWrapper) residentServiceRestClient.patchApi(
+					env.getProperty(ApiName.IDAUTHREVOKEVID.name()) + requestDto.getIndividualId(), MediaType.APPLICATION_JSON, request,
```

Comment:
Please handle the slash here before appending the individual id, since by default if you look at the configs like IDREPOGETIDBYUIN in resident-mz.properties, there is no slash at the end of config.

Reply:
we are going to work on this anyways in the upcoming story. the way we have worked on this VID is not correct. We need to rework. For now, can we go ahead with this?

Reply:
Which upcoming story we are refering here, like to know if that is part of LTS release.